vllm/entrypoints/openai/cli_args.py

```diff
@@ -93,6 +105,12 @@ def make_arg_parser():
         default=int(ssl.CERT_NONE),
         help="Whether client certificate is required (see stdlib ssl module's)"
     )
+    parser.add_argument(
+        "--privileged",
```

Comment:
Per my comments on #3237, I don't understand the argument being "--privileged" here. This should more aptly be called something like `--enable-debug-reload-route`.

Reply:
Modified to --debug argument.